ASWF/meetings/working_groups/configs/2020-11-03.md

```diff
@@ -0,0 +1,148 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+November 3, 2020
+
+Host: Michael Dolan
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Dennis Adams - Sony
+  * [X] Alessandra Tomassi - Industrial Light & Magic
+  * [X] J Schulte - Industrial Light & Magic
+
+Apologies:
+  * Carol Payne
+  * Thomas Mansencal
+
+# **OCIO Configs Working Group Meeting Notes**
+
+* Time change:
+    - Michael: All daylight savings shifts are now done and Thomas is unable to 
+      attend. Can we shift one hour earlier next time?
+    - All ok with this. We will try to not change time mid-daylight savings 
+      shift in future to reduce confusion.
+
+* Projects status:
+    - Michael: If there is needed discussion around the current generator, 
+      let's move that to Slack for now. Other code can be shared via a 
+      draft PR to share if needed.
+    - Mark: Still need to talk to ACES about naming goals, to get on same page. 
+      Alex was at last meeting. We should reach out to him. Since we're pulling 
+      naming from CTL, ideally correct in aces-dev. CTL in flux - some PRs that 
+      when merged will remove a lot of complexity. We can influence change 
+      there similarly to how ACES influences OCIO change.
+    - Michael: Last meeting we discussed that the input BuiltinTransforms were 
+      largely present, but needed output transforms to proceed. Is there an 
+      expected timeline for that work Doug?
+    - Doug: Will try to get something before next meeting.
+    - Mark: OCIO v2 has display transforms. Are we making use of that? Some 
+      transforms have OCES space, but PQ goes straight from scene linear. This 
+      config is supposed to represent all the features of OCIO v2. should we 
+      make use of both?
+    - Doug: Yes, will include this in proposal, but think will make use of 
+      display color spaces, so output transforms will be view transform plus 
+      display transform. OCES wouldn't have an interface. More recent ACES 
+      transforms have moved away from it. OCES was interesting theoretical 
+      idea, providing a break point between points of transform that proved to 
+      be more confusing. Would not use OCES.
+    - Mark: What would become the display space? The generator draws 
+      connections between the ACES blobs and the newer transforms skip the OCES 
+      step.
+    - Doug: Display telemetry. New transforms go through that but don't break 
```

Comment:
telemetry --> colorimetry
New transforms --> All the Output Transforms

Reply:
Fixed.